tests/Unit/Instances/Delete.ts

```diff
@@ -0,0 +1,34 @@
+import JsonResponse from "../../../src/Interfaces/JsonResponse"
```

Comment:
This test doesn't run. You can confirm with a coverage report or by inspecting the output in either your terminal or actions